inspire/base/templates/search/collection_authors.html

```diff
@@ -27,6 +27,13 @@
       </div>      
       {% block search_bar %}
         {{ super() }}
+        <script type="text/javascript">
```

Comment:
Since this is only temporary until the author pages go live, I've opted to use inline JS in this case.